ztp/resource-generator/Containerfile

```diff
@@ -1,5 +1,5 @@
 # Builder
-FROM openshift/origin-release:golang-1.16 as builder
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 as builder
```

Comment:
any particular reason we want to use the ci registry instead of a catalog released build?

Reply:
It's the same image that cnf-tests uses, no other reason.

Reply:
Also to use the released build I'd have to make more changes in the CI lane so it builds.

Reply:
makes sense!